save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/controller/AgentsController.kt

```diff
@@ -74,7 +76,8 @@ class AgentsController(
                 .flatMap { containerIds ->
                     agentService.saveAgentsWithInitialStatuses(
                         containerIds.map { containerId ->
-                            AgentDto(containerId, request.executionId)
+                            val containerName = dockerClient.inspectContainerCmd(containerId).exec().name
```

Comment:
need to move it to `com.saveourtool.save.orchestrator.runner.AgentRunner` (in `dockerClient`) and implement this only for `DockerAgentRunner` and return `containerId` for `KubernetersManager`

Reply:
We need to put `AgentEnvName.AGENT_NAME.name` with the same value as `AgentEnvName.AGENT_ID.name`.
I think we can add to `KubernetersManager`
```kotlin
private val agentIdEnv = EnvVar().apply {
    name = AgentEnvName.AGENT_ID.name
    valueFrom = EnvVarSource().apply {
        fieldRef = ObjectFieldSelector().apply {
            fieldPath = "metadata.name"
        }
    }
}
private val agentNameEnv = EnvVar().apply {
    name = AgentEnvName.AGENT_NAME.name
    valueFrom = agentIdEnv.valueFrom
}
```

Reply:
Thank you!